vaapi/vaapicontext.cpp

```diff
@@ -24,6 +24,7 @@
 #include "common/common_def.h"
 #include "vaapi/vaapidisplay.h"
 #include "vaapi/VaapiUtils.h"
+#include <va/va_str.h>
```

Comment:
We can't include this unconditionally since libyami supports both libva 1.0 and libva 2.0